reference/service/saml-gsuite.md

```diff
@@ -40,6 +40,7 @@ Pulumi organization were `acmecorp`, those values would be:
 
     * ACS URL: `https://api.pulumi.com/login/acmecorp/sso/saml/acs`
     * Entity ID: `https://api.pulumi.com/login/acmecorp/sso/saml/metadata`
+    * Start URL: `https://api.pulumi.com/login/acmecorp/sso`
```

Comment:
Given the way this reads, I'd probably suggest pulling it into its own line, maybe just under the image, since these two bullets and the graphic refer to the required fields, and this third field is both optional and useful enough to warrant a bit of explanation. (We used Okta at Chef, and these shortcuts came in really handy! I didn't realize Google could do something similar.) Maybe something like:

```
    ![Step 4: ...step-4.png)

   If you'd like to let your users launch the Pulumi Cloud Console from within the Google Apps menu, you can set an optional **Start URL**  as well &mdash; for example, `https://api.pulumi.com/login/acmecorp/sso`. 

   Leave the other fields ...
```

...or the like?

Reply:
+1 to having a little more copy about the IdP initiated login scenario.

If this Start URL is just POSTed to directly from the IdP, we could work around the library issue by just having it go directly to the `https://**app.pulumi.com**/welcome/acmecorp/sso` (And just treat the POST like a GET.) Do I have that right? (I like the approach you're taking of fixing the issue in the SAML library, just trying to understand the issue.)